### ISSUE TITLE
Brain.js cpu-only version

### ISSUE THREAD

**Issue by Shreni37** [BrainJS/brain.js#861]
## Summary

I need a version of brain.js where it only uses the CPU only. 

### Motivation

Hosts without a GPU cant run brain.js so we need a version of brain.js without a gpu, yk.

**Comment by imkane** [comment]
@mrtianjin829 this already exists. By default the brain functions are for CPU. The only one that's GPU-specific is `NeuralNetworkGPU()`.

**Comment by Shreni37** [comment]
> @mrtianjin829 this already exists. By default the brain functions are for CPU. The only one that's GPU-specific is `NeuralNetworkGPU()`.

Thanks for your suggestion but I was saying that i need a seperate npm package of brain.js without the GPU stuff.

**Comment by Shreni37** [comment]
@imkane Hi, is the cpu-only only version of brain.js implemented? Please let me know.

**Comment by FlowsterApp** [comment]
@mrtianjin829 I'm not a developer on this repo, just a user :slightly_smiling_face:

**Comment by imkane** [comment]
@mrtianjin829 sorry, I'm not a developer in this repo, just a user :slightly_smiling_face:

**Comment by splurgebudget** [comment]
I agree with this.  Hard to understand why there isn't a headless version of this library.  I even tried to build from scratch - but the dependencies are hardcoded to requiring GPUs.

As pointed out here, there shouldn't be a need for GPUs in order to use the non-GPU parts of the code.
